packages/@atjson/document/src/serialize.ts

```diff
@@ -307,6 +312,24 @@ export function serialize(
         edgeBehaviour,
       }
     );
+    if (annotation instanceof UnknownAnnotation) {
+      unknown.push(annotation);
+    }
+  }
+  if (options?.onUnknown && unknown.length > 0) {
+    let info = `Unknown annotations were found:\n${unknown
+      .map(
+        (annotation) =>
+          `- ${annotation.attributes.type}[${annotation.start}..${annotation.end}]`
+      )
+      .join("\n")}`;
+
+    if (options.onUnknown === "throw") {
+      throw new Error(info);
+    } else {
+      // eslint-disable-next-line
+      console.warn(info);
+    }
```

Comment:
It might make sense to document this default behavior as `onKnown: "warn" | "error" | "ignore" = "ignore`

Reply:
Ahh, yea, that may be more explicit here. Currently `undefined` and `"ignore"` are the same.